paper/paper.md

```diff
@@ -136,13 +136,25 @@ authentication) as well as processing functions for unpacking archives (zip or
 tar) and decompressing files (gzip, lzma, and bzip2).
 
 To the best of the authors' awareness, the only other Python software with some
-overlapping functionality is [Intake](https://github.com/intake/intake).
-While Intake is powerful and can be used to manage large data archives,
-we argue that Pooch has a simpler setup and meets the
-specific needs of scientific software authors and individual scientists.
-For example, Pooch does not require users to change their data loading code to
-fit into a plug-in structure, instead only providing the file path for the
-user.
+overlapping functionality are [Intake](https://github.com/intake/intake) and
+[fsspec](https://github.com/intake/filesystem_spec) (which is the backend for Intake).
+The fsspec library provides a unified interface for defining file systems and
+opening files, regardless of where the files are located (local system,
+HTTPS/FTP servers, Amazon S3, Google Cloud Storage, etc).
+fsspec implements similar download and caching functionality to
+the one in Pooch, but has a wider range of download methods available.
+In the future, fsspec could be used as a backend to expand Pooch's download
+capabilities beyond HTTPS and FTP.
+Intake manages data catalogues (with download locations and extensive
+metadata), data download and caching (using fsspec), data loading,
+visualization, and browsing.
+It has built-in capabilities for loading data into standard containers,
+including numpy, pandas, and xarray.
+While Intake and fsspec are powerful and highly configurable,
+we argue that Pooch's advantage is in its simplicity.
+Pooch has a simpler setup and is more accessible to developers without advanced
```

Comment:
Note that fsspec has no dependencies at all (but the backends do, e.g., requests for http(s)), and is probably of a similar install size to pooch. It is also available on pypi. 

I'm not sure that "more accessible" is reasonable:

Pooch:
```
GOODBOY = pooch.create(
    # specific arguments that need to be learned
)
GOODBOY.load_registry(
    pkg_resources.resource_stream("mypackage", "registry.txt")
)
fname = GOODBOY.fetch("gravity-data.csv")
data = pandas.read_csv(fname)
```

fsspec:
```
with fsspec.open('filecache://github.com/org/package/raw/version/gravity-data.csv.gz', 
                  target_protocol:'https', cache_storage='/tmp/files/') as of:
    data = pandas.read_csv(of)
```
(not that I use compression here, handled by fsspec not pandas)
(also note that we have an experimantal [github backend](https://github.com/intake/filesystem_spec/blob/master/fsspec/implementations/github.py#L6) where you can specify branch/tab/sha)

The biggest thing that is missing is the checksum and version information. I think it would be best for everyone to implement these in fsspec with directly in the existing caching implementation, or as a backend which can read/use pooch spec and/or download mechanism.

I'd like to point out that fsspec is not exactly a niche unknown package (see download stats on [pipy](https://pypistats.org/packages/fsspec) and [conda](https://anaconda.org/search?q=fsspec)).

Reply:
> I'm not sure that "more accessible" is reasonable:

The "more accessible" part should be read together with "to developers without advanced programming knowledge" but that isn't a good phrasing for what I wanted to say. I agree that the code example you posted is way nicer in fsspec than pooch. I'll try better articulate this sentence.

> The biggest thing that is missing is the checksum and version information. I think it would be best for everyone to implement these in fsspec with directly in the existing caching implementation, or as a backend which can read/use pooch spec and/or download mechanism.

OK, intake/filesystem_spec#227 makes more sense to me now :slightly_smiling_face: